packages/graphql/test/catalogPermissions.test.ts

```
@@ -25,7 +32,7 @@ describe('Unit tests covering the creation of derived permissions for catalog co
   })
 
   beforeEach(async () => {
-    await testInitialization(prisma, emitter)
+    const {} = await testInitialization(prisma, hatchet, emitter)
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)